src/components/common/modal/VideoModal.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+
+import { Modal, Button } from 'antd';
```

Comment:
Codacy found an issue: ['Button' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5801772)